### PR TITLE
amam/update_dependencies

### PR DESCRIPTION
**Pull request by cakasuma**
### Update packages
`High || Moderate vulnerability`
- grunt
- grunt-sass
- stylelint
- svgo

*Note: 
- added `node-sass` as new requirement for grunt-sass newer version `implementation sass is required to be passed on option`
-  also added new aliases `start` to skip test for development compile and serve
- there still left `tar` dependency of node-gyp error. we can wait until they release the `3.8.1`

### CODE REVIEW

File: build/aliases.yaml [hunk 1]
```diff
@@ -20,6 +20,13 @@ default:
   - 'css'
   - 'js'
 
+start:
+  - 'css'
+  - 'copy'
+  - 'shell:compile_dev'
+  - 'connect:livereload'
+  - 'concurrent:serve'
```
Comment (negarn) [inline build/aliases.yaml]: don't we also need `'js'`?

File: build/webpack/plugins.js [hunk 2]
```diff
@@ -26,11 +26,11 @@ const getPlugins = (app, grunt) => ([
...
-                cachedId                 : 'app_2',
-                dontCacheBustUrlsMatching: /\.\w{8}\./,
-                minify                   : false,
-                navigateFallback         : '',
-                filepath                 : path.resolve(PATHS.DIST, `app/${lang.toLowerCase()}/service-worker.js`),
+                cachedId                     : 'app_2',
+                dontCacheBustUrlsMatching    : /\.\w{8}\./,
+                minify                       : false,
+                navigateFallback             : '',
+                filepath                     : path.resolve(PATHS.DIST, `app/${lang.toLowerCase()}/service-worker.js`),
```
Comment (negarn) [inline build/webpack/plugins.js]: why?

File: package.json [hunk 3]
```diff
@@ -63,7 +63,7 @@
     "estraverse": "4.2.0",
     "extend": "3.0.2",
     "gettext-parser": "1.3.0",
-    "grunt": "1.0.3",
+    "grunt": "^1.0.4",
```
Comment (negarn) [inline package.json]: AFAIK we don't want to use `^`, we want to use exact version

File: package.json [hunk 4]
```diff
@@ -137,6 +138,7 @@
     "scriptjs": "2.5.8",
     "sinon": "7.2.2",
     "smartcharts-beta": "0.4.15",
+    "tar": "^4.4.8",
```
Comment (negarn) [inline package.json]: what is this for?

File: build/webpack/plugins.js (hunk 2 above)
Comment (cakasuma) [reply build/webpack/plugins.js]: my eslint gave me error, i'll revert it

File: package.json (hunk 3 above)
Comment (cakasuma) [reply package.json]: sure 👍

File: package.json (hunk 4 above)
Comment (cakasuma) [reply package.json]: that is the sub of sub dependency update, but, i think we just wait until that dependency update then we can update it. for now, ill remove it

File: build/aliases.yaml [hunk 5]
```diff
@@ -20,6 +20,14 @@ default:
   - 'css'
   - 'js'
 
+start:
+  - 'css'
+  - 'js'
+  - 'copy'
```
Comment (negarn) [inline build/aliases.yaml]: should we do copy before css and js, same as in default?

File: build/sass.js [hunk 6]
```diff
@@ -1,6 +1,9 @@
+const sass = require('node-sass');
+
 module.exports = function (grunt) {
     const options = {
         style: 'expanded',
+        implementation: sass,
```
Comment (negarn) [inline build/sass.js]: what does this do?

Comment (cakasuma) [reply build/sass.js]: the new grunt-sass needs implementation passed as an argument in the option, the sass implementation comes from node-sass dependency

**Pull request merged**